src/operator/optimizer_op.cc

```diff
@@ -259,5 +260,30 @@ to be 0.9 and the learning rate :math:`\eta` to be 0.0001.
 .add_argument("delta", "NDArray-or-Symbol", "delta")
 .add_arguments(RMSPropAlexParam::__FIELDS__());
 
+NNVM_REGISTER_OP(ftrl_update)
+.describe(R"code(Update function for Ftrl optimizer.
+Referenced from *Ad Click Prediction: a View from the Trenches*, available at
+http://dl.acm.org/citation.cfm?id=2488200.
+
+.. math::
+  \\eta_{t,i} = \\frac{learningrate}{\\beta+\\sqrt{\\sum_{s=1}^tg_{s,i}^t}}
```

Comment:
Could you explicitly state the sparse update rule like the one here in the documentation?
https://github.com/apache/incubator-mxnet/blob/master/src/operator/optimizer_op.cc#L159

Reply:
pls check again